src/annotator/integrations/index.js

```diff
@@ -15,7 +14,7 @@ import {
  * Create the integration that handles document-type specific aspects of
  * guest functionality.
  *
- * @param {Annotator} annotator
+ * @param {Pick<Annotator, 'anchor'|'anchors'>} annotator
```

Comment:
The definition of the `Annotator` type is:

```js
/**
 * Subset of the `Guest` class that is exposed to integrations.
 *
 * @typedef Annotator
 * @prop {Anchor[]} anchors
 * @prop {(ann: AnnotationData) => Promise<Anchor[]>} anchor
 */
```

It is already narrowly scoped for the purpose of use by integrations and only has these two fields, so there is no need to refine it further.